liquidations/aave-v2/index.ts

```diff
@@ -1,99 +1,102 @@
-import { gql, request } from "graphql-request"
-import { getPagedGql } from "../utils/gql"
+import { gql, request } from "graphql-request";
+import { getPagedGql } from "../utils/gql";
 
 const query = gql`
-query users($lastId: String) {
-    users(first: 1000, where:{
-        borrowedReservesCount_gt: 0,
-        id_gt: $lastId
-      }) {
-        id
-        reserves{
-          usageAsCollateralEnabledOnUser
-          reserve{
-            symbol
-            usageAsCollateralEnabled
-            underlyingAsset
-            price {
-                priceInEth
-            }
-            decimals
-            reserveLiquidationThreshold
+  query users($lastId: String) {
+    users(first: 1000, where: { borrowedReservesCount_gt: 0, id_gt: $lastId }) {
+      id
+      reserves {
+        usageAsCollateralEnabledOnUser
+        reserve {
+          symbol
+          usageAsCollateralEnabled
+          underlyingAsset
+          price {
+            priceInEth
           }
-          currentATokenBalance
-          currentTotalDebt
+          decimals
+          reserveLiquidationThreshold
         }
+        currentATokenBalance
+        currentTotalDebt
+      }
     }
-    _meta{
```

Comment:
better not to remove this, need to use it to make sure that subgraph is not stale

Reply:
its not, i added the meta bit but never got around to adding the check against current block (bug in adapter)

the way its handled on defillama adapters its by querying subgraph at a specific block, if subgraph is not synced then query just fails

i wanted to check against block here instead because liquidation data is much more time sensitive

Reply:
got it, will implement in the future in a diff pr